shared/data/model/src/main/kotlin/com/ivy/data/model/Transaction.kt

```diff
@@ -15,6 +16,7 @@ sealed interface Transaction : Syncable<TransactionId> {
     val description: NotBlankTrimmedString?
     val category: CategoryId?
     val time: Instant
+    val paidForDateTime : LocalDateTime?
```

Comment:
The domain data model must use `Instant` (time in UTC) to prevent time zone issues. Also this field doesn't seem to be a core one so move inside `TransactionMetadata`

Reply:
Sure, i was looking for how to make it specific one, so that Income, Expense etc doesn't needs to handle this override.